scalalib/src/mill/scalalib/scalafmt/ScalafmtModule.scala

```diff
@@ -0,0 +1,36 @@
+package mill.scalalib.scalafmt
+
+import ammonite.ops._
+import mill._
+import mill.define.{Command, Sources, Worker}
+import mill.scalalib._
+
+trait ScalafmtModule extends JavaModule {
+
+  def reformat(): Command[Unit] = T.command {
+    worker().reformat(
+      filesToFormat(sources()),
+      scalafmtConfig().head,
+      scalafmtDeps().map(_.path)
+    )
+  }
+
+  def scalafmtVersion: T[String] = "1.5.1"
+
+  def scalafmtConfig: Sources = T.sources(pwd / ".scalafmt.conf")
```

Comment:
Don't we have an alternative for `Sources` for a single file?

Reply:
I don't think so; we could add one if it comes up repeatedly, but for now just picking the first item in the `Agg[PathRef]` may be fine